pkg/cmd/cli/cmd/rollback.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	kubecmd "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd"
+
+	latestapi "github.com/openshift/origin/pkg/api/latest"
+	client "github.com/openshift/origin/pkg/cmd/client"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+)
+
+const rollbackLongDesc = `
+Rolls back a deploymentConfig to match a previous deployment. By default, only the pod
+template of the specified deployment will be included in the rollback.
+
+Examples:
+  $ %[1]s %[2]s --from=deployment-2
```

Comment:
Why isn't this just an arg?  I can't imagine ever rolling back something that isn't a deployment...